examples/twittermap/web/public/javascripts/app.js

```diff
@@ -1,4 +1,5 @@
-var app = angular.module('cloudberry', ['cloudberry.map','cloudberry.timeseries','cloudberry.sidebar', 'cloudberry.util','cloudberry.cache']);
+var app = angular.module('cloudberry', ['cloudberry.map','cloudberry.timeseries','cloudberry.sidebar', 'cloudberry.util','cloudberry.cache',
+                          'cloudberry.mapresultcache']);
```

Comment:
we may want to cut the line more balanced?

Reply:
I shall modify it.